src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagReporter.java

```diff
@@ -33,19 +34,34 @@ public static void initializeBugsnagErrorReporting() {
     /**
      * If Bugsnag has been configured, report error based on provided information.
      */
-    public static boolean reportErrorToBugsnag(Throwable throwable, String message) {
+    public static boolean reportErrorToBugsnag(String message, Throwable throwable) {
+        return reportErrorToBugsnag(message, null, throwable);
+    }
+
+    /**
+     * If Bugsnag has been configured, report error based on provided information.
+     */
+    public static boolean reportErrorToBugsnag(String message, Object badEntity, Throwable throwable) {
+        LOG.error(message, throwable);
         if (bugsnag == null) {
-            LOG.warn("Bugsnag error reporting is disabled. Unable to report this message: {} ", message, throwable);
+            LOG.warn("Bugsnag error reporting is disabled. Unable to report to Bugsnag in this context: {}, this message: {}",
+                message,
+                badEntity,
+                throwable);
             return false;
         }
 
         if (throwable == null) {
-            LOG.warn("An exception is mandatory with Bugsnag error reporting. Unable to report this message: {} ", message);
+            LOG.warn("This error is not an exception and will not/cannot be reported to Bugsnag in this context: {}, this message: {}",
+                message,
+                badEntity);
```

Comment:
Is the order for `message` and `context` swapped?

Reply:
Updated.